Form-Controls/style.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
```

Comment:
I like the use of relative units. What is the 100vh doing in this style? What happens when you take it away?